xray/core/dataarray.py

```diff
@@ -1091,6 +1091,16 @@ def plot(self):
         '''
         return _PlotMethods(self)
 
+    def _title_for_slice(self):
```

Comment:
Was waffling around where to put it. It's been in 3 different places over the course of this PR- here, in utils.py, and in the plot module. If you think the plot module is the best place then I'll put it there.